app/controllers/teacher_interface/new_regs/work_histories_controller.rb

```diff
@@ -0,0 +1,283 @@
+# frozen_string_literal: true
+
+module TeacherInterface
+  module NewRegs
```

Comment:
`NewRegulations`? Just thinking we don't generally abbreviate except for the pesky acronym ones.

Reply:
I see this is in a few places. Given they'll be the only regs in a couple of months I can live with the abbreviation.

Reply:
I wasn't particular happy with the module anyway, but it seemed like the only option. I'm happy to change it, or leave it, either way.

Reply:
Let's leave it. I'll do a card for removing the old regs 'in the future ™️'  and then this will just be regs 🤣